tests/routes/test_keys.py

```
@@ -73,3 +76,16 @@ def test_create_key_with_expires_at(
         "/v2/keys", params={"workspace_id": test_workspace.name}
     )
     assert response.status_code == 401
+
+
+def test_create_key_auth_disabled(client: TestClient, monkeypatch: pytest.MonkeyPatch):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_create_key_auth_disabled`

Add return type annotation: `None`

(ANN201)